src/App.js

```diff
@@ -1,10 +1,37 @@
-import React from 'react';
+import React from "react";
+import Tile from "./common/Tile";
+import Poster from "./assets/images/poster.svg";
 
 function App() {
+  const movieDetails = {
+    title: "Mulan",
+    poster: Poster,
+    year: 2020,
+    genres: ["action", "drama", "adventure"],
+    description:
+      "A young Chinese maiden disguises herself as a male warrior in order to save her father. Disguises herself as a male warrior in order to save her father. A young Chinese maiden disguises herself as a male warrior in order to save her father.",
+    production: "China, USA",
+    release: "24.10.2020",
+    rate: 7.8,
+    votes: 35,
+  };
+
   return (
-    <div>
-      Strona główna...
-    </div>
+    <>
+      <div>Strona główna...</div>
+      <Tile
+        horizontal
+        title={movieDetails.title}
+        poster={movieDetails.poster}
+        year={movieDetails.year}
+        description={movieDetails.description}
+        production={movieDetails.production}
+        release={movieDetails.release}
+        genres={movieDetails.genres}
+        rate={movieDetails.rate}
+        votes={movieDetails.votes}
+      />
+    </>
```

Comment:
Tu będzie konflikt z routerem, który jest już zmergowany, więc proponuję test tego kafelka przenieść do MoviesPage.

Reply:
Teraz jak już Router jest na mainie to można tak zrobić ale konfilikt błachy, wystarczy to dać pod nav a przed w Switch i zgrać ze sobą importy. To w moment do ogarnięcia nawet przez strone gita także luz :)

Reply:
Ok, tak to zrobię ;)